polling_stations/apps/data_collection/management/commands/import_west_lancashire.py

```diff
@@ -67,20 +62,32 @@ def address_record_to_dict(self, record):
             "100010684612",  # WN80AG -> WN80AH : 7 Ormskirk Road, Up Holland, Skelmersdale
             "100012654494",  # L397HS -> L397HQ : Rose Croft, Broad Lane, Downholland, Ormskirk
             "100012821133",  # WN88AU -> WN80PB : 21 Liverpool Road, Skelmersdale
-            "10012363326",  # L392EJ -> L392DE : 1000 Walmsley Drive, Ormskirk
-            "100010675944",  # WN88PU -> WN88EN : 83B School Lane, Skelmersdale
             "100012657542",  # L407SN -> L407SL : The Homestead Merridale Farm, High Lane, Ormskirk
             "100012415199",  # L407SN -> L407SL : The Mount, High Lane, Ormskirk
-            "10012341150",  # L407SN -> L407GA : 6 Blythe Meadow, High Lane, Ormskirk
-            "10012341151",  # L407SN -> L407GA : 7 Blythe Meadow, High Lane, Ormskirk
             "10012356919",  # WN80HS -> WN89QS : 93 Dearden Way, Up Holland, Skelmersdale
             "10012344656",  # L406JA -> L406JT : Ottershead Barn, Dicks Lane, Lathom, Ormskirk
             "100012414799",  # L406JA -> L406JT : Whitestone Cottage, Dicks Lane, Lathom, Ormskirk
             "100012418034",  # WN69EN -> WN69EQ : Glen View, Hall Lane, Wrightington, Wigan
             "100010677297",  # WN80EW -> WN80EN : Hallbridge Farmhouse, 59 Dingle Road, Up Holland, Skelmersdale
             "100010682656",  # WN80QY -> WN80QZ : Pearsons Farm, Lafford Lane, Up Holland, Skelmersdale
-            "100010667268",  # L397EA -> L394QW : 245 St Helens Road, Lathom Skelmersdale
+            "100012657092",  # L405SW -> L405UZ : Lyme House, Junction Lane, Burscough, Ormskirk
+            "10012357601",  # L393BH -> L393BW : 14 Park Road, Ormskirk
+            # Just not sure
```

Comment:
Cool. It would be good if there were a way to say "I agree with the not-sure-what-to-do-here", so I can squash the warning while having the default behaviour.

Reply:
Now removed from the list anyway :-)